app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -3,75 +3,105 @@
 
 $navbar-default-bg: $primary_color;
 
-@import "bootstrap";
+.fact-col{
```

Comment:
Why is this here? This file should only contain overrides to the default patternfly css. 
Any css that is not related to overriding the defaults should either go:
a. in case it applies to every/most pages - in application.scss
b. in case it applies to a specific page - in <page>.scss, scoped under an id selector inside that page if relevant.